tests/test_makeps3iso.py

```
@@ -148,6 +149,156 @@ async def test_plan_job_rejects_output_outside_volumes(tmp_path, monkeypatch):
+            str(link_root) + suffix,
+            spec=registry.spec("folder_to_iso"),
+            mode="folder_to_iso",
+            output_dir=None,
+            duplicate_action=convert_routes.DuplicateAction.SKIP,
+            delete_on_verify=False,
+        )
+    assert exc.value.reason is convert_routes.SkipReason.PS3_FOLDER_UNSAFE
+
+
+@pytest.mark.parametrize(
+    "raw, expected",
```

Ruff (0.15.20)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)